source/configure-connection.txt

```diff
@@ -0,0 +1,175 @@
+.. _django-connection-configuration:
+
+==================================
+Configure Your Database Connection
+==================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect your Django project to MongoDB
+and configure the connection.
+
+Connection Configuration 
+------------------------
+
+After installing {+django-odm+} and creating a project, you can configure
+your connection to MongoDB in the following ways:
+
+- :ref:`django-connection-configure-manual` by specifying the
+  ``DATABASES`` variable in your project's settings
+- :ref:`django-connection-configure-automatic` by using
+  the ``parse_uri()`` method
+
+.. tip::
+
+   To learn how to install the Django ODM and create a
+   Django project, visit the :ref:`django-get-started` tutorial.
+
+.. _django-connection-configure-manual:
+
+Manually Configure Database Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To manually configure your connection to MongoDB, update
+the ``DATABASES`` variable in your project's ``settings.py``
+file. Set the ``DATABASES`` variable to a dictionary value containing
+the database settings. You must configure the dictionary's
+``default`` key to set MongoDB as the default database
+connection.
+
+To configure the ``default`` key, assign a nested dictionary as its value.
+This nested dictionary has the following keys:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Key
+     - Description
+
+   * - **ENGINE**
+     - The backend driver to use for the connection. Set this key to ``"django_mongodb_backend"``.
+
+   * - **HOST**
+     - | Your connection URI. For localhost connections, this key is optional. 
+       | For SRV connections, you must include a scheme prefix (``mongodb+srv://``).
+       |
+       | If connecting to a replica set or sharded cluster with multiple hosts, specify 
+       | each host separated by a comma.
```

Comment:
s: slight wording suggestion. could also be helpful to include an example, either here or later.

```suggestion
       | To specify more than one host, include all hostnames in one string. Use
       | a comma to separate each hostname.
```